includes/options/class-amp-options-menu.php

```diff
@@ -633,6 +647,40 @@ public function render_caching() {
 		<?php
 	}
 
+	/**
+	 * Render the stories export settings section.
+	 *
+	 * @since 1.2
+	 */
+	public function render_stories_export() {
+		?>
+		<style>
+			label[for="story_export_base_url"] span {
+				text-transform: uppercase;
+				font-size: 0.7em;
+				border: 1px solid;
+				border-radius: 2px;
+				padding: 2px;
+				margin: -15px 0  0 3px;
+				position: relative;
+				top: -2px;
+				font-weight: 400;
+				line-height: 1;
+			}
+		</style>
+		<fieldset <?php disabled( ! current_user_can( 'edit_files' ) ); ?>>
+			<p>
+				<label for="story_export_base_url">
+					<strong><?php echo wp_kses_post( __( 'Base URL for exported stories. <span>Beta</span>', 'amp' ) ); ?></strong>
```

Comment:
Like @swissspidy [mentioned](https://github.com/ampproject/amp-wp/pull/2625#issuecomment-509644664), it looks like if this isn't populated:
<img width="659" alt="not-populated" src="https://user-images.githubusercontent.com/4063887/60923025-a37f4600-a263-11e9-8173-83ef660bfab6.png">

...the exported file looks to have an empty `assets/` directory, and I didn't see a notice or warning of this:

<img width="603" alt="assets-dir-empty" src="https://user-images.githubusercontent.com/4063887/60922964-8b0f2b80-a263-11e9-83c2-f304a7edb9ae.png">